docs/REST API/rest-api.md

```diff
@@ -5827,10 +5829,85 @@ Deletes the custom MDM setup enrollment profile assigned to a team or no team.
 `Status: 204`
 
 
+### Get Over-the-Air (OTA) enrollment profile
+
+`GET /api/v1/fleet/enrollment_profiles/ota`
+
+The returned value is a signed `.mobileconfig` OTA enrollment profile. Install this profile on macOS, iOS, or iPadOS hosts to enroll them to a specific team in Fleet and turn on MDM features.
+
+To turn enroll macOS hosts, turn on MDM features, and add [human-device mapping](#get-human-device-mapping), install the [manual enrollment profile](#get-manual-enrollment-profile) instead.
```

Comment:
@noahtalerman should this be either "turn on" or "enroll"? Or is "turn enroll" some special terminology?

Reply:
Whoops! That's a typo. This should be "enroll." I fixed it.